lib/connect.js

```diff
@@ -76,6 +76,8 @@ let didPrintDscpTaggingWarning = false;
  *   room.once('disconnected', function() {
  *     console.log('You left the Room:', room.name);
  *   });
+ * }).catch(error => {
+ *   console.log('Could not connect to the room: ' + error.message);
```

Comment:
@makarandp0 Nitpick:
```suggestion
 *   console.log('Could not connect to the Room:', error.message);
```